### ISSUE TITLE
Incorrect annotation when cancelling an export task

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#6251]
### Observed behavior
Although cancelling an export task is possible, compared to delete tasks (see #6250), the values indicated seem to suggest that the entire channel has been exported, despite cancelling almost immediately. 

![2019-12-09_8-25-18](https://user-images.githubusercontent.com/1457929/70416198-c6a26180-1a5e-11ea-9b47-0251fe972b79.png)

Above I first imported the full PhET channel, proceeded to start the export of all the resources to the attached drive, and cancelled after 2 seconds. I have serious doubts that time was sufficient to export all 238 MB.

### Expected behavior
Correctly reported number and size of exported resources after cancelling the task.

### User-facing consequences
Unmet expectations regarding the number of exported resources.

### Errors and logs
…

### Steps to reproduce
Start an export task, cancel and observe the number of resources reported as exported.

### Context
 * Kolibri version: 0.13.0b4
 * Operating system: Windows 7
 * Browser: both Chrome & FF

**Comment by jonboiser** [comment]
@rtibbles Added a new piece of metadata to handle canceled 'importcontent' tasks here (the `transferred_resources` field) and is supposed to be used to show how much was actually transferred in case of a cancelation or failure.

https://github.com/learningequality/kolibri/blob/05e6f190aadc30afc6a041318e87ef25bfe9aebf/kolibri/core/content/management/commands/importcontent.py#L311-L313

I'm noticing something similar is not in the 'exportcontent'. Maybe that's why you're not getting similar "partial task" messages for export.

**Comment by rtibbles** [comment]
I didn't add this for the export task, because in that case, cancellation triggers disk cleanup (i.e. no files will have been exported after the task has been cancelled) - but this means there is differential behaviour that either the frontend or the backend needs to handle.

**Comment by indirectlylit** [comment]
canceling a deletion seems like unnecessary and confusing functionality. Would it help to intentionally hide 'cancel' button during deletion?

**Comment by radinamatic** [comment]
This issue is about cancelling en **Export** task 🙂

**Comment by radinamatic** [comment]
And why would canceling a deletion be confusing or unnecessary, users can realize an error or change their minds about what to do all the time. I'd say cancelling a delete operation is valuable (Gmail gives you several seconds to undo a delete).

**Comment by jonboiser** [comment]
@rtibbles Since cancelled exports clean up after themselves, would it be more accurate to say "0 of X resources exported" if an export is canceled? Should we literally show that message in that situation?

**Comment by rtibbles** [comment]
I feel like just saying "cancelled" or some such is probably simpler, and just skipping the summary completely.

**Comment by jonboiser** [comment]
That works for me. We already have a 'canceled' label on the top of the card, so it could be like this

```
Canceled
Export 'channel name'
<nothing here>
Started by 'jb'
```

**Comment by rtibbles** [comment]
Fixed in #6300
